.github/workflows/unit-test.yml

```diff
@@ -74,24 +74,24 @@ jobs:
       RUN_MYSQL_TESTS: 1
       RUN_POSTGRES_TESTS: 1
       RUN_REDIS_TESTS: 1
-      OPENTELEMETRY_MEMCACHED_HOST: localhost
-      OPENTELEMETRY_MEMCACHED_PORT: 11211
-      POSTGRES_USER: postgres
-      POSTGRES_DB: circle_database
-      POSTGRES_HOST: localhost
-      POSTGRES_PORT: 5432
-      OPENTELEMETRY_REDIS_HOST: localhost
-      OPENTELEMETRY_REDIS_PORT: 6379
+      CASSANDRA_HOST: localhost
       MONGODB_DB: opentelemetry-tests
       MONGODB_HOST: localhost
       MONGODB_PORT: 27017
-      MYSQL_USER: otel
-      MYSQL_PASSWORD: secret
       MYSQL_DATABASE: circle_database
       MYSQL_HOST: localhost
+      MYSQL_PASSWORD: secret
       MYSQL_PORT: 3306
+      MYSQL_USER: otel
+      OPENTELEMETRY_MEMCACHED_HOST: localhost
+      OPENTELEMETRY_MEMCACHED_PORT: 11211
+      OPENTELEMETRY_REDIS_HOST: localhost
+      OPENTELEMETRY_REDIS_PORT: 6379
+      POSTGRES_DB: circle_database
+      POSTGRES_HOST: localhost
+      POSTGRES_PORT: 5432
+      POSTGRES_USER: postgres
```

Comment:
Is this actually changed or just sorted?

Reply:
Just sorted. Mentioned that in the OP.